tikv/tikv
# tikv-ctl: Migrate clap to structopt

### What problem does this PR solve?

Issue Number: close #10793

What's Changed:

Add `structopt` as a dependency, for parsing command line options of `tikv-ctl`.

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
  - I tried my best to ensure CLI behaviour keep the same after the migration. Most codes are transformed literally.
  - The `--help` output for every subcommand is printed and compared between clap version and structopt version, so that we can ensure we have the same number of arguments/flags with flag types unchanged.
  - I tested most command in the [`tikv-ctl` part of TiKV documention](https://tikv.org/docs/dev/reference/tools/tikv-ctl/) manually. But supported flags are too many. I am not able to test every possible flag/argument combination to ensure correctness.

### Release note 

```release-note
None
```
